### PR TITLE
add GOVERNANCE.md and OWNERS

### PR DESCRIPTION
**Pull request by Iceber**
Signed-off-by: Iceber Gu <wei.cai-nat@daocloud.io>

**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
+# Clusterpedia Project Governance
+## Membership
```
Comment (pacoxu) [inline GOVERNANCE.md:2]: sig node 文档里，这个叫 contributor ladder。
https://github.com/kubernetes/community/pull/6725/files 这个写的挺好的，可以瞅一眼。

**Comment by wuyingjun-lucky** [comment]
looks good to me 
@hanweisen @xyz2277  Can you help to review ?

**Comment by Iceber** [comment]
> 例如 Primary reviewer for at least 10 substantial PRs to the codebase 会进行加粗，这样可能更直观和直接

我分别对关键词和关键句子加粗，你觉得那种更清晰一些呢

* member for at least **1 months**
* Primary reviewer for at least **3 PRs** to the codebase
* Reviewed or merged at least **10 substantial PRs** to the codebase
* Knowledgeable about the codebase
* Sponsored by a [submodel](#submodel-and-owners-files) approver
    * With no objections from other approvers
    * Done through PR to update the [OWNERS](#submodel-and-owners-files) file
* May either self-nominate, be nominated by an approver in this [submodel](#submodel-and-owners-files).

or

* **member for at least 1 months**
* **Primary reviewer for at least 3 PRs to the codebase**
* **Reviewed or merged at least 10 substantial PRs to the codebase**
* Knowledgeable about the codebase
* Sponsored by a [submodel](#submodel-and-owners-files) approver
    * With no objections from other approvers
    * Done through PR to update the [OWNERS](#submodel-and-owners-files) file
* May either self-nominate, be nominated by an approver in this [submodel](#submodel-and-owners-files).

看起来好像加粗句子好像更好一些

**Comment by wuyingjun-lucky** [comment]
> > 例如 Primary reviewer for at least 10 substantial PRs to the codebase 会进行加粗，这样可能更直观和直接
> 
> @wuyingjun-lucky 我分别对关键词和关键句子加粗，你觉得那种更清晰一些呢
> 
> * member for at least **1 months**
> * Primary reviewer for at least **3 PRs** to the codebase
> * Reviewed or merged at least **10 substantial PRs** to the codebase
> * Knowledgeable about the codebase
> * Sponsored by a [submodel](#submodel-and-owners-files) approver
>   
>   * With no objections from other approvers
>   * Done through PR to update the [OWNERS](#submodel-and-owners-files) file
> * May either self-nominate, be nominated by an approver in this [submodel](#submodel-and-owners-files).
> 
> or
> 
> * **member for at least 1 months**
> * **Primary reviewer for at least 3 PRs to the codebase**
> * **Reviewed or merged at least 10 substantial PRs to the codebase**
> * Knowledgeable about the codebase
> * Sponsored by a [submodel](#submodel-and-owners-files) approver
>   
>   * With no objections from other approvers
>   * Done through PR to update the [OWNERS](#submodel-and-owners-files) file
> * May either self-nominate, be nominated by an approver in this [submodel](#submodel-and-owners-files).
> 
> 看起来好像加粗句子好像更好一些

如果我选择的话 我会选择 下面

**Comment by Iceber** [comment]
@wuyingjun-lucky 更新了， PTAL

**Comment by wuyingjun-lucky** [comment]
> @wuyingjun-lucky 更新了， PTAL

我觉得OK，语法问题不擅长，但是整体描述 我能一眼看懂，包括每个角色的要求以及责任。

Looks good to me

**Comment by wzshiming** [comment]
The root directory of the repo requires a default OWNERS file.

**Comment by hanweisen** [comment]
> looks good to me @hanweisen @xyz2277 Can you help to review ?

looks good to me

**Comment by Iceber** [comment]
> The root directory of the repo requires a default OWNERS file.

okay

@wzshiming Do we need to do anything with the bot other than update the https://github.com/clusterpedia-io/gh-ci-bot?

**Comment by wuyingjun-lucky** [comment]
Maybe a bug @wzshiming  
New commits is updated but the lgtm label is not canceled

**Comment by Iceber** [comment]
@wuyingjun-lucky You can create a pr to add yourself to the OWNERS of the module you want to join, and you can add the core developers of memory storage as reviewers

**Comment by wzshiming** [comment]
> Maybe a bug @wzshiming New commits is updated but the lgtm label is not canceled

The ci-bot will only work on the comment, not on the code push, 
If necessary, we need to add a new gh-action to remove the lgtm label when a new push.

**Pull request merged**